app/src/main/java/com/duckduckgo/app/statistics/VariantManager.kt

```diff
@@ -36,7 +40,15 @@ interface VariantManager {
 
             // Shared control. You can use this as your control unless you are experimenting on
             // a subgroup e.g a device API or specific language
-            Variant(key = "sc", weight = 1.0, features = emptyList())
+            Variant(key = "sc", weight = 1.0, features = emptyList()),
+
+            // Notification Content
+            Variant(key = "me", weight = 1.0, features = listOf(NotificationPrivacyDay1, NotificationSuppressClearDataDay3)),
+            Variant(key = "mi", weight = 1.0, features = listOf(NotificationClearDataDay1, NotificationSuppressClearDataDay3)),
+
+            // Notification Add New
+            Variant(key = "mf", weight = 1.0, features = listOf()),
```

Comment:
I assume `mf` is needed but i can't immediately see why `sc` wouldn't suffice?

Reply:
Good shout, it should suppress the clear data notification in prod, I've updated it.